File: Python/DayOne/simple_arithmetic_app.py
import random
import datetime
 
start = datetime.datetime.now()
count_correct = 0
for _ in range(10):
	first_number = random.randint(0,100)
	second_number = random.randint(0,100)

	minimum = min(first_number,second_number)
	maximum = max(first_number,second_number)
	answer = maximum - minimum

	
	user_input = float(input("What is " + str(maximum) + " - " + str(minimum) + "?: "))

	if user_input == answer:
		count_correct += 1

	if user_input != answer:
		user_input_2 = float(input("Try again: "))

		if user_input_2 != answer:
			print("The correct answer is: " + str(answer))


print("You answered " + str(count_correct) + " questions correctly out of 10 questions!")
end = datetime.datetime.now()
print("Time taken: " + str(end - start))

File: Java/DayOne/SimpleArithmeticApp.java
import java.util.Scanner;
public class SimpleArithmeticApp {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

int countCorrect = 0;
for(int count = 0; count < 10; count++) {
	Double firstNumber = Math.floor(Math.random()  * 100);
	Double secondNumber = Math.floor(Math.random() * 100);
	
	Double maximum = Math.max(firstNumber,secondNumber);
	Double minimum = Math.min(firstNumber,secondNumber);

	Double answer = maximum - minimum;
	
	System.out.print("What is " + maximum + " minus " + minimum + ": ");
	int userInput = input.nextInt();
	
	if(userInput == answer) {
		countCorrect += 1;
	}
	
	if(userInput != answer) {
		System.out.print("Try again: ");
		int userInputTwo = input.nextInt();
		if(userInputTwo != answer) {
			System.out.println("The correct answer is : " + answer);
		}
	}
	
}
System.out.print("You answered " + countCorrect + " questions out of 10 questions correctly!");

}
}